### ISSUE TITLE
Coturn in Ubuntu 16.04 FUBAR; no connection from NAT'ed clients and logs to debug

### ISSUE THREAD

**Issue by ivdok** [coturn/coturn#304]
Config:
```
listening-port=80
tls-listening-port=443
alt-listening-port=3478
alt-tls-listening-port=5349
verbose
fingerprint
no-auth
realm=realm
log-file=/var/www/html/coturn.log
syslog
simple-log
no-cli
external-ip=_MYEXTERNALIP_
listening-ip=_MYEXTERNALIP_
relay-ip=_MYEXTERNALIP_

```
Systemd unit:
```[Unit]
Description=coturn
Documentation=man:coturn(1) man:turnadmin(1) man:turnserver(1)
After=syslog.target network.target

[Service]
Type=simple
User=root
Group=root
EnvironmentFile=/etc/default/coturn
PIDFile=/run/turnserver/turnserver.pid
ExecStart=/usr/bin/turnserver --daemon --syslog --pidfile /run/turnserver/turnserver.pid -c /etc/turnserver.conf $EXTRA_OPTIONS
Restart=on-abort
LimitCORE=infinity
LimitNOFILE=1000000
LimitNPROC=60000
LimitRTPRIO=infinity
LimitRTTIME=7000000
CPUSchedulingPolicy=other
UMask=0007

[Install]
WantedBy=multi-user.target
```
Package version 4.5.0.7-1ubuntu2.
For whatever fucking reason it doesn't do what it should, and there's no way to even debug.

**Comment by misi** [comment]
Can you share the logs?
Cert is installed?

**Comment by ivdok** [comment]
TLS is neither used nor with certificate. The logs got pruned, but I've solved the issue on my own. It was a weird combination of several mistakes:
- `lt-cred-mech` was used in conjuction with `use-auth-secret` and `static-auth-secret` blindly in accordance with guides on the 'net. However, it caused conflict between REST API and credential auth, as [you've stated here.](https://github.com/coturn/coturn/issues/180#issuecomment-364363272) Wish I found this earlier though, not buried deep within google search. Solved by disabling REST altogether.
- SQLite path confusion in Ubuntu package, where the actual DB location is `/var/lib/turn/turndb`, but the default, empty config expects `/var/db/turndb` or `/usr/local/var/db/turndb` or whatever, solved by manually specifying path in config.

Still got issues with stability (sometimes it works, sometimes it returns 401 Unauthorized even with valid tokens, usually solved by stopping unit, recreating users with turnadmin, flushing FS caches and restarting again) and simultaneous syslog/simple log write, but these are irrelevant to the ticket discussed, and it may be closed.
